src/main.js

```diff
@@ -7,13 +7,27 @@ import App from './App.vue'
 import router from './router'
 import store from './store';
 import BootstrapVue from 'bootstrap-vue'
+import VueApollo from 'vue-apollo'
+import ApolloClient from 'apollo-boost'
 
 Vue.config.productionTip = false
 Vue.use(BootstrapVue)
 Vue.use(Vuex)
+Vue.use(VueApollo)
+
+
+const apolloClient = new ApolloClient({
+  // You should use an absolute URL here
+  uri: 'http://localhost:3100/v2'
```

Comment:
Looks like this overrides the value in the other config file.  Should it be based on the config file or specified here?  This appears to be the correct URL here.